charts/akeyless-secure-remote-access/README.md

```diff
@@ -57,6 +57,7 @@ helm install RELEASE_NAME akeyless/akeyless-sra -f values.yaml
 | `apiGatewayURL`                    | API Gateway URL to use to fetch the secrets **Required**                                                                      | `https://rest.akeyless.io`
 | `privilegedAccess`                    | Supported auth methods: AWS IAM, Azure AD, GCP and API Key.For AWS IAM or Azure AD, or GCP provide only accessID. For API Key, provide both accessID and accessKey                                                                       | ` `
 | `legacySigningAlg`                 | When set to "true", will sign ssh certificates using the legacy 'ssh-rsa-cert-v01@openssh.com' signing algorithm name in the certificate. | `"false"`
+| `usernameSubClaim`                 | When set, it will extract the username from the provided claim key-name, and use it to authenticated to the remote target (RDP or SSH) | ` `
```

Comment:
Sorry for the multiple revisions, but let's use this version:

usernameSubClaim  | Optional, provide a key-name to extract the username (value) of that key in sub-claims. The extracted username will be used to authenticated to the remote target (RDP or SSH).

Reply:
Fixed